node-operators/networks/collator.md

```diff
@@ -96,75 +112,62 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
-
- 8. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+  1. 进入**Developer**标签，点击**Extrinsics**
+  2. 选择您想用于参与收集活动的账户
+  3. 确认您的收集人账户已充值[所需的最低质押量](#accounts-and-staking-requirements)，并有多余金额用于支付交易费
+  4. 在**submit the following extrinsic**菜单中选择**parachainStaking** pallet
+  5. 打开下拉菜单，在质押相关的所有外部参数中，选择**joinCandidates()**函数
+  6. 将绑定数量设置为成为候选人的[最低数量](#accounts-and-staking-requirements)（输入数量需以`wei`为单位）。例如，在Moonbase Alpha的最低绑定数量为{{ networks.moonbase.staking.min_can_stk }}枚DEV，以wei为单位应输入`{{ networks.moonbase.staking.min_can_stk_wei }}`（即{{ networks.moonbase.staking.min_can_stk }}再额外加18个0）。在Moonriver的最低绑定数量为{{ networks.moonriver.staking.min_can_stk }}枚MOVR，以wei为单位应输入`{{ networks.moonriver.staking.min_can_stk_wei }}`（即{{ networks.moonriver.staking.min_can_stk }}再额外加18个0）。这里仅考虑候选人的绑定数量，其他委托量将不计入统计
+  7. 设置候选人数量即候选人池大小。如何设置该数值请查看[获取候选人池的大小](#get-the-size-of-the-candidate-pool)部分
+   8. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Join candidate pool via Polkadot.js](/images/node-operators/networks/collators/collator-polkadotjs-16.png)
 
 !!! 注意事项
     函数名称和最低绑定数量要求可能会在未来发布新版本时有所调整。
 
-如上所述，只有在Moonbase Alpha上委托质押量最高的前{{ networks.moonbase.staking.max_collators }}名候选人和Mooriver上委托质押量最高的前{{ networks.moonriver.staking.max_collators }}名候选人才可以进入收集人有效集。
+如上所述，只有质押量排名靠前的收集人才可以进入收集人有效集。每个网络候选人的具体排名数量，请查看本教程的[账户与质押要求](#accounts-and-staking-requirements)部分。
 
-## 停止参与收集活动 {: #stop-collating }
+## 停止参与收集活动 {: #stop-collating } 
 
 在最新的runtime升级（[runtime version 1001](https://moonbeam.network/announcements/staking-changes-moonriver-runtime-upgrade/)）中，用户与质押功能的交互方式进行了重大升级，其中包含取消质押的方式。
 
-想要停止参与收集活动并离开候选人池，您必须先发起请求。发起请求并不会自动将您从候选人池中移除，您需要等待延迟时段，才能够执行请求并停止参与收集活动。在Moonriver上会有{{ networks.moonriver.collator_timings.leave_candidates.rounds }}轮（{{ networks.moonriver.collator_timings.leave_candidates.hours }}小时）的延迟。在Moonbase Alpha上会有{{ networks.moonbase.collator_timings.leave_candidates.rounds }}（{{ networks.moonbase.collator_timings.leave_candidates.hours }}小时）的延迟。在等待具体的轮次数的过程中，如果您在有效集中您仍有资格生产区块和获取奖励。
+想要停止参与收集活动并离开候选人池，您必须先发起请求。发起请求并不会自动将您从候选人池中移除，您需要等待[退出生效期](#collator-timings)，才能够执行请求并停止参与收集活动。在等待具体轮次数的过程中，如果您在有效集中您仍有资格生产区块和获取奖励。
 
-#### 发起离开候选人的请求 {: #schedule-request-to-leave-candidates }
+### 发起离开候选人的请求
 
 执行以下步骤开始发起一个请求：
 
  1. 进入**Developer**标签
+  2. 点击**Extrinsics**
 
- 2. 点击**Extrinsics**
-
- 3. 选择您的候选人账户
+  3. 选择您的候选人账户
 
- 4. 在**submit the following extrinsic**菜单中选择**parachainStaking** pallet
+  4. 在**submit the following extrinsic**菜单中选择**parachainStaking** pallet
 
- 5. 选择**scheduleLeaveCandidates**函数
+  5. 选择**scheduleLeaveCandidates** extrinsic
 
- 6. 输入`candidateCount`（可从[获取候选人池的大小](#get-the-size-of-the-candidate-pool)部分获得）
+  6. 输入`candidateCount`（可从[获取候选人池的大小](#get-the-size-of-the-candidate-pool)部分获得）
 
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+   7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Schedule leave candidates request](/images/node-operators/networks/collators/collator-polkadotjs-9.png)
 
-#### 执行离开候选人的请求 {: #execute-request-to-leave-candidates }
+### 执行离开候选人的请求
```

Comment:
this is missing header attribute, should be `{: #execute-request-to-leave-candidates }` still